Userland/Utilities/file.cpp

```diff
@@ -230,7 +231,8 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         } else {
             auto file_name_guess = Core::guess_mime_type_based_on_filename(path);
             auto mime_type = Core::guess_mime_type_based_on_sniffed_bytes(*file).value_or(file_name_guess);
-            auto human_readable_description = get_description_from_mime_type(mime_type, path).value_or(mime_type);
+            auto fallback = TRY(String::from_utf8(mime_type));
+            auto human_readable_description = TRY(get_description_from_mime_type(mime_type, path)).value_or(fallback);
```

Comment:
You can avoid always allocating the fallback if we just make this not use `value_or`:
```c++
auto human_readable_description = TRY(get_description_from_mime_type(mime_type, path));
if (!human_readable_description.has_value())
    human_readable_description = TRY(String::from_utf8(mime_type));
outln("{}: {}", path, flag_mime_only ? mime_type : *human_readable_description);
```

Reply:
Good idea! Fixed.